storage/redis.js

```diff
@@ -214,47 +219,100 @@ class StorageRedis extends StorageInterface {
   }
 
   /**
-   * listen to redis events for expired and deleted keys
+   * TODO DOC!!!
+   * options.invalidation.clear.minInterval min interval between two clear operations, default 1 sec
+   * options.invalidation.clear.maxInterval min interval between two clear operations, default 10 sec
+   * options.invalidation.clear.maxTime heuristic max time to full clear the dangling references
    */
-  async listen (listener) {
-    if (!listener) {
-      this.log.warn({ msg: 'acd/storage/redis.listen, listener connection is missing' })
+  heuristicClean({ start = false, interval, lastCursor, retrieves = 10 } = {}) {
```

Comment:
@mcollina that's the idea after out last talk

wdyt? I still have to write test and so on, but it should work

https://github.com/mcollina/async-cache-dedupe/blob/7519e18edc771eb3022be28345ec009b5084d223/storage/redis.js#L227